ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -192,7 +197,9 @@ public static Object toFieldType(final Expression expressionValue) {
   /**
   * Determines the type of command a sql string is and returns a SqlCommand.
   */
+  // CHECKSTYLE_RULES.OFF: CyclomaticComplexity
```

Comment:
Rather than disabling this checkstyle rule, I think we'd have better readability by pulling out the logic for each case in this method into separate helper methods.